release/microProfile1/java8/ibmjava/Dockerfile

```diff
@@ -2,3 +2,4 @@ FROM open-liberty:kernel-java8-ibm
 
 COPY server.xml /config/
 
+RUN /opt/ol/wlp/bin/server start && /opt/ol/wlp/bin/server stop && rm -rf /output/resources/security/
```

Comment:
I'm not super familiar with the whole docker image, so I'm assuming  /output/resources/security simply gets populated with some data that we'd like to discard.

Reply:
Yes, it contains ltpa keys and may contain ssl certificates.